inst/scripts/run_sctour.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+import sctour as sct
+import scanpy as sc
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import json
+
+def run_sctour(GEXfile, metafile, exclusion_json_path, ptime_out_file):
+    adataObj = sc.read_10x_h5(GEXfile)
+    info = pd.read_csv(metafile, sep=',', index_col=0)
+    with open(exclusion_json_path) as f:
+        exclusionList = json.load(f)
+    cells = adataObj.obs_names.intersection(info.index)
+    if len(cells)==0:
+        print("No cells in intersection of adata object and metadata supplied in info. Please ensure these objects are correct.")
+        exit(1)
+    adataObj = adataObj[cells, :]
+    adataObj.obs['ClusterNames_0.2'] = info.loc[cells, 'ClusterNames_0.2'].copy()
+    adataObj.obs['SubjectId'] = info.loc[cells, 'SubjectId'].copy()
+    adataObj.shape
+    adataObj.X = round(adataObj.X).astype(np.float32)
+    adataObj.obs['Population'] = info.loc[cells, 'Population'].copy()
```

Comment:
It seems like these variable should be somehow parameterized, and not hard coded?